napari/settings/_application.py

```diff
@@ -155,6 +155,13 @@ class ApplicationSettings(EventedModel):
         title=trans._("Grid Height"),
         description=trans._("Number of rows in the grid."),
     )
+    confirm_close_window: bool = Field(
+        default=True,
+        title=trans._("Window close confirmation"),
+        description=trans._(
+            "Ask for confirmation before close window with Ctrl+W"
```

Comment:
might be worth also native-ifying the shortcut here?

Reply:
[642ff14](https://github.com/napari/napari/pull/4637/commits/642ff14e54e8b04cd06f32554d1db5b1ff7cb43f)

Reply:
Okay good to know - we should probably have a simple python library which does this somewhere then but we can leave it for this PR